### DISCUSSION TITLE
v3.0.0-beta

### DISCUSSION THREAD

**Discussion by ahmad-moussawi** [sqlkata/querybuilder#626] [Announcements]
## What's Changed
* Fixes empty identifier issue by @gbrunton in https://github.com/sqlkata/querybuilder/pull/467
* Issue 182 - delete join support by @ahmad-moussawi in https://github.com/sqlkata/querybuilder/pull/614
* Close issues after 30days if no reply by @ahmad-moussawi in https://github.com/sqlkata/querybuilder/pull/615
* Fix places where the parameterPlaceholder is hardcoded to a ?  by @toburger in https://github.com/sqlkata/querybuilder/pull/616
* Iss 507 support filter clause by @ahmad-moussawi in https://github.com/sqlkata/querybuilder/pull/617
* Iss 581 where exists remove select by @ahmad-moussawi in https://github.com/sqlkata/querybuilder/pull/618
* Fix for Oracle insert many by @mathijs-dumon in https://github.com/sqlkata/querybuilder/pull/557

## New Contributors
* @gbrunton made their first contribution in https://github.com/sqlkata/querybuilder/pull/467
* @toburger made their first contribution in https://github.com/sqlkata/querybuilder/pull/616

**Full Changelog**: https://github.com/sqlkata/querybuilder/compare/v2.4.0...v3.0.0-beta

<hr /><em>This discussion was created from the release <a href='https://github.com/sqlkata/querybuilder/releases/tag/v3.0.0-beta'>v3.0.0-beta</a>.</em>

**Comment by asherber** [comment]
@ahmad-moussawi I note that this is a major version bump. Are there breaking changes?

**Reply by ahmad-moussawi** [reply]
Hello, my intention was that this version contains breaking changes, but it seems not :)
I have some PR that needs to be merged containing BC, in the mean time I will wait for feedback for the current release.
and I will follow the semantic versioning, since previously SqlKata was not really following the semantic version approach, so bumping to 3.x.x may be a good start
